apps/component-generator/component-templates/ComponentTemplate/package.json

```diff
@@ -22,12 +22,7 @@
     "url": "https://github.com/microsoft/fluentui-react-native.git",
     "directory": "packages/components/ComponentTemplate"
   },
-  "dependencies": {
-    "@fluentui-react-native/framework": "0.7.17",
-    "@fluentui-react-native/text": ">=0.12.1 <1.0.0",
-    "@fluentui-react-native/tokens": ">=0.11.8 <1.0.0",
-    "@fluentui-react-native/use-styling": ">=0.8.2 <1.0.0"
-  },
+  "dependencies": {},
```

Comment:
Intended?

Reply:
Yes, I want to make sure people are making sure they're using the up to date versions of these packages. The versions that were in the input's package.json were the same as those listed here, causing the issue since they;re very out of date.